app/models/page.rb

```diff
@@ -45,7 +45,7 @@ def breadcrumbs
     list = [self, self&.parent, self&.parent&.parent, self&.parent&.parent&.parent].compact.reverse
     list.shift
     list.each_with_object({ 'Home' => '/' }) do |obj, memo|
-      memo[obj.title] = obj.path
```

Comment:
This code looks like `#footer_path`, is there a DRY-er way of doing this?

Reply:
I have addressed this in https://github.com/DFE-Digital/early-years-foundation-reform/pull/781